### ISSUE TITLE
Make sure the i18n is set up and displayed in all projects

### ISSUE THREAD

**Issue by jeremyboles** [centralstandard/folamh#26]
I think that a few of the projects might be missing i18n. This should be added before we declare this project a _set up_. 😉
